ansible/group_vars/all

```diff
@@ -530,9 +530,9 @@ scheduler:
     maxRetriesToGetQueue: "{{ scheduler_maxRetriesToGetQueue | default(13) }}"
   queue:
     # the queue's state Running timeout, e.g. if have no activation comes into queue when Running, the queue state will be changed from Running to Idle and delete the decision algorithm actor
-    idleGrace: "{{ scheduler_queue_idleGrace | default('20 seconds') }}"
+    idleGrace: "{{ scheduler_queue_idleGrace | default('10 minutes') }}"
```

Comment:
As warmed containers are kept for 10 minutes, it's better to keep the queue as well.
It's relatively safer than containers to keep an idle queue as it uses fewer resources when idle.